From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aditya Prayoga <aditya@kobol.io>
Date: Tue, 15 Sep 2020 20:04:22 +0700
Subject: arm64: dts: rockchip: helios64: fan cooling properties

#cooling-cells plus cooling-min-state/cooling-max-state on both fans.
No cooling-maps reference the fans; the min/max-state properties have
no binding.

Upstream: none.

Split out of add-board-helios64.patch (Aditya Prayoga, rebased onto
mainline by rpardini for 6.3); the DTB content is unchanged.

Signed-off-by: Aditya Prayoga <aditya@kobol.io>
Assisted-by: LLM
Signed-off-by: Igor Velkov <iav@iav.lv>
---
 arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
@@ -78,6 +78,9 @@ / {
 		/* fan connected to P7 */
 		compatible = "pwm-fan";
 		pwms = <&pwm0 0 40000 0>;
+		cooling-min-state = <0>;
+		cooling-max-state = <3>;
+		#cooling-cells = <2>;
 		cooling-levels = <0 80 170 255>;
 	};
 
@@ -85,6 +88,9 @@ / {
 		/* fan connected to P6 */
 		compatible = "pwm-fan";
 		pwms = <&pwm1 0 40000 0>;
+		cooling-min-state = <0>;
+		cooling-max-state = <3>;
+		#cooling-cells = <2>;
 		cooling-levels = <0 80 170 255>;
 	};
 
-- 
Armbian

